test.py

```diff
@@ -711,22 +727,25 @@ class BoostTest(UnitTest):
 
     def __init__(self, test_no: int, shortname: str, suite, args: str,
                  casename: Optional[str], allows_compaction_groups : bool) -> None:
-        boost_args = []
         if casename:
             shortname += '.' + casename
-            boost_args += ['--run_test=' + casename]
         super().__init__(test_no, shortname, suite, args)
```

Comment:
`init` uses `casename` in `_get_args` which is not set yes

Reply:
thanks. fixed.